README.rst

```diff
@@ -215,7 +215,7 @@ database connections at ``fork``.
 Documentation
 =============
 
-The `latest documentation`_ is hosted at Read The Docs, containing user guides,
+The `latest documentation`_ is hosted at reading The Docs, containing user guides,
```

Comment:
The name of the service is indeed `Read the Docs`: 
- https://readthedocs.org/projects/celery/
- https://celery.readthedocs.io/en/latest/